stack.yaml

```diff
@@ -1,11 +1,9 @@
-resolver: lts-9.0
+resolver: lts-9.6
 extra-deps:
   - configurator-ng-0.0.0.1
   - critbit-0.2.0.0
-  - hasql-pool-0.4.1
   - hjsonpointer-1.1.1
   - hjsonschema-1.5.0.1
-  - jose-0.5.0.3
```

Comment:
Nice to be able to remove one of these custom deps

Reply:
I am using `nix` which uses stack LTS by default. It is a bit annoying when an older version is used which motivates me for the change.